midi_generator/wandb-sweep.yaml

```diff
@@ -0,0 +1,13 @@
+program: /project/midi_generator/train.py
```

Comment:
do we even need hypertrain for this (and other) recipes? we should either adjust it and document, or drop.

Reply:
Nope, it think it is not needed here. Because train.py does not support `make hyperparameters-search`

Reply:
It tried to say that it's broken functionality. We have two possible ways to deal with it:
1) Add support in `train.py` of mentioned in this file parameters
2) Drop this file and `make hyperparameters-tuning` command.